cmd/kaf/config.go

```diff
@@ -161,6 +162,35 @@ var configAddClusterCmd = &cobra.Command{
 	},
 }
 
+var configRemoveClusterCmd = &cobra.Command{
+   	Use:   "remove-cluster [NAME]",
+    Short: "remove cluster",
+    Args:  cobra.ExactArgs(1),
+    Run: func(cmd *cobra.Command, args []string) {
+        name := args[0]
+
+        var pos = -1
+        for i, cluster := range cfg.Clusters {
+            if cluster.Name == name {
+                pos = i
+                break
+            }
+        }
+
+        if pos == -1 {
+            errorExit("Could not delete cluster: cluster with name '%v' not exists.", name)
+        }
+
+        cfg.Clusters = append(cfg.Clusters[:pos],cfg.Clusters[pos+1:]...)
+
+        err := cfg.Write()
+        if err != nil {
+            errorExit("Unable to write config: %v\n", err)
+        }
+        fmt.Println("Remove cluster.")
```

Comment:
It should be "Removed cluster."

Reply:
ok, I have modified it according to your suggestion